### DISCUSSION TITLE
When will the latest version of dp gen offline be available?

### DISCUSSION THREAD

**Discussion by 13793005587** [deepmodeling/dpgen#1652] [Q&A] [answered]
Dear developers,
Because my research group's server is not connected to the external network, we can only install the offline version, which has resulted in some features being unusable. So I would like to ask when the offline version can be updated?

**Comment by njzjz** [comment, marked as answer]
We will not update the offline version, but you can follow [How to install conda/pip packages without the Internet access](https://docs.deepmodeling.com/faq/conda.html#how-to-install-conda-pip-packages-without-the-internet-access).

**Reply by 13793005587** [reply]
Thank you for your help！
